test/powershell/engine/ETS/Adapter.Tests.ps1

```diff
@@ -1,143 +1,228 @@
 Describe "Adapter Tests" -tags "CI" {
-    BeforeAll {
-        $pso  = [System.Diagnostics.Process]::GetCurrentProcess()
-        $processName = $pso.Name
-
-        if(-not ('TestCodeMethodClass' -as "type"))
-        {
-            class TestCodeMethodClass {
-                static [int] TestCodeMethod([PSObject] $target, [int] $i)
-                {
-                    return 1;
+    Context "Property Adapter Tests" {
+        BeforeAll {
+            $pso  = [System.Diagnostics.Process]::GetCurrentProcess()
+            $processName = $pso.Name
+
+            if(-not ('TestCodeMethodClass' -as "type"))
+            {
+                class TestCodeMethodClass {
+                    static [int] TestCodeMethod([PSObject] $target, [int] $i)
+                    {
+                        return 1;
```

Comment:
Wouldn't it be better to return `$i` instead of `1`?

Reply:
I don't know. It is used only in one test and the test works well.